.changeset/swift-fans-protect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': minor
```

Comment:
Should be `patch` since there are no breaking changes in the APIs?

Reply:
You’re right, I will change that